### ISSUE TITLE
its not working after the update today ?

### ISSUE THREAD

**Issue by samehnp** [fmmmlee/GW2-Addon-Manager#102]
when i start the game and when character window appears .. it always get me a msg tht there is an error happned to the game and when i close it it closes the game ... is it a bug or there is something should i do b4 the luanch ? ( i try to restart the UAOM but still got the same problem when i remove it game works correctly  and iam only using the 12dx prxy )

**Comment by mirel442** [comment]
same problem crash when game start

**Comment by fantomshard** [comment]
the manger dose not even start for me any more

**Comment by ThiagoScala** [comment]
same problem here and for some reason i can't unninstall the addons via manager. nothing happens when i do it.

**Comment by Ghostkat9** [comment]
There have been updates to the dx912pxy addon as well which haven't been installable through the Addon Manager for quite a while. I know the argument has been regarding the GW2Radial plugin that it was in alpha and was a pre-release version, but it looks almost as if the plugin has been abandoned by its maker as people have asked on that Github what's happening with it, and the developer has been silent on the subject.

**Comment by Draeggiar** [comment]
I dont think this is and issue with Addon manager. After GW2 path addons may not work until creators update them as well (especialy d912pxy). Just disable them in addon manager, and try updating them after a few days, works for me every time.
